locales/locales.php

```diff
@@ -462,6 +462,18 @@ public function __construct() {
 		$ca->google_code = 'ca';
 		$ca->facebook_locale = 'ca_ES';
 
+		$ca_valencia = new GP_Locale();
+		$ca_valencia->english_name = 'Catalan (Valencian)';
+		$ca_valencia->native_name = 'Català (Valencià)';
+		$ca_valencia->lang_code_iso_639_1 = 'ca';
+		$ca_valencia->lang_code_iso_639_2 = 'cat';
+		$ca_valencia->wp_locale = 'ca_valencia';
+		$ca_valencia->slug = 'ca-valencia';
```

Comment:
@ocean90 , it seems that slug used to be `ca/valencia` instead of the one I added here (`ca-valencia`). 

https://meta.trac.wordpress.org/changeset/9839

Happy to have that changed too